README.md

```diff
@@ -106,28 +106,29 @@ AP and inject the configuration into the image (see
 for more details), or use a configuration for an ethernet connection:
 
 ```shell
-$ balena os configure balena-debug.img --app DebugApp --config-network=ethernet
+$ balena os configure balena-debug.img --fleet DebugApp --config-network=ethernet
```

Comment:
```suggestion
$ balena os configure balena-debug.img --fleet DebugFleet --config-network=ethernet
```

Reply:
@vipulgupta2048 I wonder if I can find and replace DebugApp -> DebugFleet, even log entries to convert the DebugApp to DebugFleet. I'll have to find slugs too that have ryanh/debugapp for example. I'll have to be careful, but that way we can phase out App. Hmm.